lib-injection/runner/action.yml

```diff
@@ -22,6 +22,7 @@ runs:
         with:
           repository: 'DataDog/system-tests'
           path: './system-tests'
+          ref: tonycthsu/fix-ruby-images-for-lib-injection-tests
```

Comment:
Is this change still needed? E.g. should it be merged to master? (It seems like something for testing...?)